R/fullEDGEBUILDINGS.R

```diff
@@ -0,0 +1,41 @@
+#' fullEDGEBUILDINGS
+#'
+#' Function that produces the complete ISO data set required for the
+#' EDGE - Buildings model.
+#'
+#' @param rev data revision which should be used as input (positive numeric).
+#'
+#' @author Antoine Levesque, Robin Krekeler
+#'
+#' @seealso
+#' \code{\link{readSource}},\code{\link{getCalculations}},\code{\link{calcOutput}}
+#' @examples
+#' \dontrun{
+#' fullEDGEBUILDINGS()
+#' }
+#'
+#' @importFrom madrat calcOutput
+#' @export
+
+fullEDGEBUILDINGS <- function(rev = 0) {
+
+  # socio-economic data --------------------------------------------------------
+  calcOutput("PopulationPast", aggregate = FALSE, file = "f_poppast.cs4r")
+  calcOutput("Population",     aggregate = FALSE, file = "f_pop.cs4r", FiveYearSteps = FALSE)
+  calcOutput("GDP",            aggregate = FALSE, file = "f_gdp.cs4r", FiveYearSteps = FALSE)
+  calcOutput("GDPPast",        aggregate = FALSE, file = "f_gdppast.cs4r")
+  calcOutput("RatioPPP2MER",   aggregate = FALSE, file = "f_ppp2mer.cs4r")
+  calcOutput("Surface",        aggregate = FALSE, file = "f_surface.cs4r")
+
+  # energy ---------------------------------------------------------------------
+  # move calcIO to mrcommons
+  calcOutput("IO", subtype = "output_EDGE_buildings", aggregate = FALSE, file = "f_edge_buildings.cs4r")
+  calcOutput("IO", subtype = "output_EDGE",           aggregate = FALSE, file = "f_edge_stationary.cs4r")
+  calcOutput("IEAPFU",                                aggregate = FALSE, file = "f_iea_pfu.cs4r")
+  calcOutput("FloorspacePast",                        aggregate = FALSE, file = "f_floorspace.cs4r")
+
+  # climate data ---------------------------------------------------------------
+  for (tlim in as.character(17:25)) {
```

Comment:
Is `as.character` required by madrat? The default arg and documentation of `calcHDDCDD` suggest `tlimit` should be an integer.

Reply:
You are right, I removed it.